docs/csharp/programming-guide/classes-and-structs/snippets/how-to-define-constants/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+static class Constants
+{
+    public const double Pi = 3.14159;
+    public const int SpeedOfLight = 300000; // km per sec.
+}
+
+class Program
+{
+    static void Main()
+    {
+        double radius = 5.3;
+        double area = Constants.Pi * (radius * radius);
+        int secsFromSun = 149476000 / Constants.SpeedOfLight; // in km
```

Comment:
Was it intentional to divide integers this way? I kept it unchanged, but I feel it's better to cast one or both operands to a double and make `secsFromSun` as double.

Reply:
Yes, it looks like this was intentional. Let's leave it as is.